apis/Google.Cloud.Bigtable.V2/Google.Cloud.Bigtable.V2/BigtableMutateRowsRequestManager.cs

```diff
@@ -153,6 +153,14 @@ internal ProcessingStatus OnOk()
                 }
                 if (_retryableCodes.Contains(status.Code))
                 {
+                    // Skip non-idempotent entries. We could check this at the other continue above, but since
```

Comment:
I think I need to look at *all* of this more carefully - I'm losing the plot of how it all hangs together.